File: ce/snippets/visualbasic/CRMV8/entitiesvb/vb/createupdateentitymetadata8.vb
' <snippetcreateupdateentitymetadata8>


 Dim BankAccountEntity As EntityMetadata = retrieveBankAccountEntityResponse.EntityMetadata

 ' Disable Mail merge
 BankAccountEntity.IsMailMergeEnabled = New BooleanManagedProperty(False)
 ' Enable Notes
 Dim updateBankAccountRequest As UpdateEntityRequest = New UpdateEntityRequest With {
  .Entity = BankAccountEntity,
  .HasNotes = True
 }
 _serviceProxy.Execute(updateBankAccountRequest)

' </snippetcreateupdateentitymetadata8>

File: ce/snippets/csharp/CRMV8/entities/cs/createupdateentitymetadata8.cs
// <snippetcreateupdateentitymetadata8>


 EntityMetadata BankAccountEntity = retrieveBankAccountEntityResponse.EntityMetadata;

 // Disable Mail merge
 BankAccountEntity.IsMailMergeEnabled = new BooleanManagedProperty(false);
 // Enable Notes
 UpdateEntityRequest updateBankAccountRequest = new UpdateEntityRequest
 {
  Entity = BankAccountEntity,
  HasNotes = true
 };



 _serviceProxy.Execute(updateBankAccountRequest);

// </snippetcreateupdateentitymetadata8>